config/environments/test.rb

```diff
@@ -46,7 +46,7 @@ module OptimizerWrapper
 
   PARAMS_LIMIT = { points: 150, vehicles: 10 }.freeze
   QUOTAS = [{ daily: 100000, monthly: 1000000, yearly: 10000000 }] # Only taken into account if REDIS_COUNT
-  REDIS_COUNT = Redis.new # Fake redis
+  REDIS_COUNT = ENV['REDIS_COUNT_HOST'] && Redis.new(host: ENV['REDIS_COUNT_HOST']) || Redis.new
```

Comment:
Here is the trick. When launching inside docker, the api service does not have a local redis because it uses a redis service.